addons/mail/static/src/models/thread.js

```diff
@@ -1945,15 +1941,11 @@ registerModel({
             compute: '_computeInvitationLink',
         }),
         /**
-         * List of guests that have been invited to the RTC call of this channel.
-         * FIXME should be simplified if we have the mail.channel.partner model
-         * in which case the two following fields should be a single relation to that model.
-         */
-        invitedGuests: many('Guest'),
-        /**
-         * List of partners that have been invited to the RTC call of this channel.
+         * List of members that have been invited to the RTC call of this channel.
          */
-        invitedPartners: many('Partner'),
+        invitedMembers: many('ChannelMember', {
+            isCausal: true,
```

Comment:
no isCausal here either, member shouldn't be removed from channel when no longer invited